editor/tasks/leiningen/pack.clj

```diff
@@ -12,13 +12,13 @@
 ;; these can be sourced either from a local build of engine, or downloaded from an archived build on s3
 (def engine-artifacts
   {"x86_64-darwin" {"bin" ["dmengine" "dmengine_release"]
-                    "lib" ["libparticle_shared.dylib" "libtexc_shared.dylib"]}
+                    "lib" ["libparticle_shared.dylib"]}
    "x86-win32"     {"bin" ["dmengine.exe" "dmengine_release.exe"]
```

Comment:
I have removed the 32 bit copying in our private repo.
That change is incoming within the week.

Reply:
Nice! That's outside of the scope of this PR, though, right?